config/schemas/config.yaml

```diff
@@ -920,6 +920,28 @@ $defs:
   cronSchedule:
     type: string
     pattern: ^(((\*\/)?([0-5]?[0-9])((\,|\-|\/)([0-5]?[0-9]))*|\*)[^\S\r\n]+((\*\/)?((2[0-3]|1[0-9]|[0-9]|00))((\,|\-|\/)(2[0-3]|1[0-9]|[0-9]|00))*|\*)[^\S\r\n]+((\*\/)?([1-9]|[12][0-9]|3[01])((\,|\-|\/)([1-9]|[12][0-9]|3[01]))*|\*)[^\S\r\n]+((\*\/)?([1-9]|1[0-2])((\,|\-|\/)([1-9]|1[0-2]))*|\*|(jan|feb|mar|apr|may|jun|jul|aug|sep|oct|nov|dec))[^\S\r\n]+((\*\/)?[0-6]((\,|\-|\/)[0-6])*|\*|00|(sun|mon|tue|wed|thu|fri|sat)))$|^@(annually|yearly|monthly|weekly|daily|hourly|reboot)$
+  alertUpstreamRunbook:
+    title: Runbooks for alerts
+    type: object
+    description: |-
+      Example:
+
+      group: link-to-alert-group-runbook
+      AlertName: link-to-specific-alert-runbook
```

Comment:
Define this object, also you should move the $def under runbookUrls because it is not going to be used anywhere else.

Reply:
> Define this object

I don't think we should define the object (each alert), to avoid default config clutter.

Or am I missing the point?

Reply:
You should define `group` as a string and define it, then define alerts under `additionalProperties` also as a string.

Reply:
Do you want me to define every single alert?

Reply:
> then define alerts under additionalProperties also as a string.

i.e. like this

``` yaml
additionalProperties:
  type: string
```

Reply:
Okay I see, should hopefully be right now https://github.com/elastisys/compliantkubernetes-apps/pull/2438/commits/c18c424239b250a022d74fd4183d5b0a8d75bdc7

Reply:
I think it looks correct now :+1: